gobblin-test-harness/src/main/java/gobblin/test/setup/config/Step.java

```diff
@@ -0,0 +1,31 @@
+package gobblin.test.setup.config;
+
+import java.util.Collection; 
+
+import gobblin.test.execution.operator.Operator;
+
+/**
+ * This interface is to define individual steps associated to a config entry, this implements the Operator interface.
+ * Essentially one step can have many operators and each operator has an execution
+ * 
+ * @author sveerama
+ *
+ */
+
+public interface Step extends Operator
```

Comment:
Can you document in more detail how this interface exactly gets used. I don't fully understand the flow from the `ConfigReader` to the `ConfigStepsGenerator` to a `Step`.